src/core/main.go

```diff
@@ -295,6 +296,7 @@ func main() {
 	middlewares := []beego.MiddleWare{
 		requestid.Middleware(),
 		orm.Middleware(legacyAPISkipper),
+		transaction.Middleware(legacyAPISkipper),
```

Comment:
Shouldn't the orm and transaction be in a same middleware?

Reply:
When orm middleware is not applied for the request but the transaction middleware is applied, the transaction middleware will be failed. I think this is expected result.